yarn.lock

```diff
@@ -24879,8 +24879,8 @@ __metadata:
 
 "ansi-color@file:./forks/ansi-color::locator=highlight%40workspace%3A.":
   version: 0.0.0
-  resolution: "ansi-color@file:./forks/ansi-color#./forks/ansi-color::hash=291f6b&locator=highlight%40workspace%3A."
-  checksum: ebc4eb866ee788f9088fd981ccec71a0aca50f8b7f798a6ab31e628b917e4554f6bc5f4c9f8119622901e59350b9bead56c69076d8b09af812c3e9111a10547a
```

Comment:
@anthonygregis Is this related? Can we revert this change if not, since it looks like it is causing an error in the build